docs/docs-ref-autogen/excel/excel.yml

```diff
@@ -176,12 +177,14 @@ classes:
   - 'excel!Excel.ChartBorder:class'
   - 'excel!Excel.ChartBoxwhiskerOptions:class'
   - 'excel!Excel.ChartCollection:class'
+  - 'excel!Excel.ChartCollectionCustom:class'
```

Comment:
These custom classes seem to have been added. Let's make sure to remove them from DefinitelyTyped.

Reply:
Good catch. I'll make a note of them!